qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -2252,13 +2248,22 @@ class Kubernetes implements OrchestratorMain {
 
     String createNamespace(String ns) {
         return evaluateWithRetry(2, 3) {
-            Namespace namespace = new Namespace("v1", null, new ObjectMeta(name: ns), null, null)
+            Namespace namespace = newNamespace(ns)
             def namespaceId = client.namespaces().createOrReplace(namespace).metadata.getUid()
             defaultPspForNamespace(ns)
             return namespaceId
         }
     }
 
+    static Namespace newNamespace(String ns) {
+        Namespace namespace = new Namespace()
+        ObjectMeta meta = new ObjectMeta()
+        meta.setNamespace(ns)
+        meta.setName(ns)
```

Comment:
I have no idea why this is working and `new Namespace("v1", null, new ObjectMeta(name: ns, namespace: ns), null, null)` not